src/Dropdown/index.js

```diff
@@ -90,6 +90,7 @@ const Dropdown = (props) => {
         field={props.field}
         label={props.triggerLabel}
         readOnly
+        error={props.error}
```

Comment:
I'd recommend adding a PropType for the `error` prop; it will show up in the props table and throw helpful errors if someone passes something other than a string

Reply:
👍